src/google/protobuf/compiler/cpp/cpp_helpers.h

```diff
@@ -774,10 +774,10 @@ class PROTOC_EXPORT NamespaceOpener {
   std::vector<std::string> name_stack_;
 };
 
-enum Utf8CheckMode {
-  STRICT = 0,  // Parsing will fail if non UTF-8 data is in string fields.
-  VERIFY = 1,  // Only log an error but parsing will succeed.
-  NONE = 2,    // No UTF-8 check.
+enum class Utf8CheckMode {
+  Strict = 0,  // Parsing will fail if non UTF-8 data is in string fields.
```

Comment:
Looks great to me but one small thing: our [style](https://google.github.io/styleguide/cppguide.html#Enumerator_Names) for enum names is to name them `kStrict`, `kVerify`, `kNone`. (Sometimes we don't use that convention in our existing code because our previous convention was to use `ALL_CAPS` names.)

Reply:
Fixed!